6.3.a_long_cat_is_long.py

```diff
@@ -0,0 +1,22 @@
+
+def count_words(text):
+    """
+    A function that takes a string of text as input and returns a dictionary
+    containing the count of each word in the text and the length of the word.
+
+    Args:
+    text (str): A string of text
+
+    Returns:
+    dict_count (dict): A dictionary containing the count of each word in the text and
+                       the length of the word.
+
+    """
+    x = ',.:;!?'

```

Comment:
x don't cover all of the possible signs that can appear beside letters.
Think of a better method to filter out these signs.